calcom/cal.com
# feat: mailhog fixture

## What does this PR do?
This PR is a followup on these PRs, #10521 and #10637.
This PR is a PoC for mailhog that can be used to assert emails.

## Type of change



- [x] Chore/Test (refactoring code, technical debt, workflow improvements)

## How should this be tested?
- Make sure the `mailhog` container is running. Ideally, it should run with`yarn dx`. 
- The default env variables should work `EMAIL_SERVER_HOST='localhost'` and `EMAIL_SERVER_PORT=1025`.
- Test `booking-pages.e2e.ts`
